src/util/metaData.ts

```diff
@@ -1,4 +1,7 @@
 import AjvModule from 'ajv'
+import type { KeywordCxt } from 'ajv'
+import { byGrapheme } from 'split-by-grapheme'
+import * as logger from '../Logger.js'
```

Comment:
'logger' is defined but never used

This is a finding by GitHubActions. There are a few other issues reported by GitHubActions and I believe that some of them should be fixed.

Reply:
yes, fixed.